src/sentry/replays/usecases/ingest/issue_creation.py

```diff
@@ -32,6 +33,23 @@ def report_rage_click_issue_with_replay_event(
 
     selector = selector
     clicked_element = selector.split(" > ")[-1]
+    component_name = component_name
+    evidence = [
+        IssueEvidence(name="Clicked Element", value=clicked_element, important=True),
+        IssueEvidence(name="Selector Path", value=selector, important=True),
+    ]
+    evidence_data = {
+        # RRWeb node data of clicked element.
+        "node": node,
+        # CSS selector path to clicked element.
+        "selector": selector,
+    }
+
+    if component_name:
+        evidence.append(
+            IssueEvidence(name="React Component Name", value=component_name, important=True),
+        )
+        evidence_data.update({"component_name": component_name})
```

Comment:
reading through the issue platform docs, https://develop.sentry.dev/issue-platform/#issue-evidence-schema,

only one piece of evidence should be marked as important. maybe we could cascade and say if we have component name, mark that as important, if not, selector path? and then mark clicked element as not important?

Reply:
good catch, i'll update